src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityWrappers.cs

```diff
@@ -301,6 +302,13 @@ private static int AccessibilityGetCursorOffsetWrapper(IntPtr self)
             return GetViewFromRefObject(self).AccessibilityGetCursorOffset();
         }
 
+        private static IntPtr AccessibilityGetRangeExtentsWrapper(IntPtr self, int startOffset, int endOffset, int coordType)
+        {
+            Rectangle rect = GetViewFromRefObject(self).AccessibilityGetRangeExtents(startOffset, endOffset, (AccessibilityCoordinateType)coordType);
+
+            return rect.SwigCPtr.Handle;
```

Comment:
Oh, I think we need to make a copy and change this line to `return DuplicateRectangle(rect);` and define `DuplicateRectangle` next to `DuplicateString` and `DuplicateAccessibilityRange` in this file. This is because C# garbage collector could theoretically free `rect` (and is guaranteed to do it with `using` above), which is a local variable, before returning to native code (and there would be a double free anyway with the current version of dali-csharp-binder).

```c#
private static IntPtr DuplicateRectangle(Rectangle rect)
{
    return Interop.Rectangle.NewRectangle(rect.X, rect.Y, rect.Width, rect.Height);
}
```

Then, `rect` (both C# and C++ object) would be freed by C# and its copy (only C++ object) would be freed by `StealObject` in `NUIViewAccessible`.

Reply:
Thank you for kind review.
I did not want to increase Duplicate* methods with poor understand.